build/DependencyVersions.props

```diff
@@ -18,7 +18,7 @@
     <CLI_NuGet_Version>4.4.0-preview3-4475</CLI_NuGet_Version>
     <CLI_NETStandardLibraryNETFrameworkVersion>2.0.0-preview3-25514-04</CLI_NETStandardLibraryNETFrameworkVersion>
     <CLI_WEBSDK_Version>2.0.0-rel-20170908-653</CLI_WEBSDK_Version>
-    <CLI_TestPlatform_Version>15.3.0-preview-20170628-02</CLI_TestPlatform_Version>
+    <CLI_TestPlatform_Version>15.5.0-preview-20170923-02</CLI_TestPlatform_Version>
```

Comment:
Should this be a final version: 15.5.0 or a preview one, as it is?

Reply:
We plan to remove the preview tag as part of branding changes in a couple of days before 15.5 release.

Reply:
I would like to get the CLI ready into VS by Wednesday.

Reply:
@livarcocc : Is this the last check-in which will go in VS? I am asking this because we will take a branding change at the end.